### PR TITLE
doc(blog): Elasticsearch blog

### PR DESCRIPTION
**Pull request by hermanschaaf**
Elasticsearch destination blog post

### CODE REVIEW

File: website/pages/blog/search-through-cloud-infrastructure-with-elasticsearch.mdx [hunk 1]
```diff
@@ -0,0 +1,81 @@
...
+
+## Introduction
+
+Have you ever wished you could enter the ID of an EC2 instance into the search bar and get a list of all the related resources? Or search by tag across all your accounts and regions? Or search for resources by keyword? With the new CloudQuery Elasticsearch destination, you can do all this, and more.
+
+Using the Elasticsearch destination, you can search through your cloud infrastructure, create visualizations and dashboards, and generally explore your infrastructure data in a way that wasn't possible before. In this article we will show you how to get started and a few examples of the things you can do.
+
+## Why Elasticsearch?
+
+Elasticsearch is a popular open source search engine built on top of Lucene. It is a distributed, RESTful search and analytics engine that is capable of storing and searching terabytes of data. As datasets grow over time, managing a SQL database can become more and more difficult, but Elasticsearch offers a great alternative here. It is schemaless and has built-in support for scaling, sharding and replication. Its fulltext search capabilities also make it a great fit for searching through unstructured data, especially JSON columns.
```
Comment (hermanschaaf) [inline website/pages/blog/search-through-cloud-infrastructure-with-elasticsearch.mdx:25]:
```suggestion
Elasticsearch is a popular open source search engine built on top of Lucene. It is a distributed, RESTful search and analytics engine that is capable of storing and searching terabytes of data. As datasets grow over time, managing a SQL database can become more and more difficult, but Elasticsearch offers a great alternative here. It is schemaless and has built-in support for scaling, sharding and replication. Its full text search capabilities also make it a great fit for searching through unstructured data, especially JSON columns.
```

File: website/pages/blog/search-through-cloud-infrastructure-with-elasticsearch.mdx [hunk 2]
```diff
@@ -0,0 +1,81 @@
...
+
+## Getting Started
+
+First, you will need to have CloudQuery installed. If you haven't already, you can do so by following the instructions in the [Quickstart](/docs/quickstart) guide.
+
+You will also need a running Elasticsearch instance. If you don't have one, you can either create one locally using Docker, or use the [Elastic Cloud](https://www.elastic.co/cloud) service, which offers a free trial. This [`docker-compose` file](https://github.com/cloudquery/cloudquery/blob/main/plugins/destination/elasticsearch/docker-compose.yaml) will get you started with local Elasticsearch and Kibana services (download the file, then run `docker compose up`). Once it's up, you should be able to visit `localhost:5601` and see the Kibana interface.
+
+## Configuring the Elasticsearch Destination
+
+Now that you have CloudQuery and Elasticsearch running, you can configure the Elasticsearch destination. To do this, follow the instructions in the [Elasticsearch Destination guide](/docs/plugins/destinations/elasticsearch/overview) to create a configuration file called `elasticsearch.yml`.
```
Comment (erezrokah) [inline website/pages/blog/search-through-cloud-infrastructure-with-elasticsearch.mdx:35]:
```suggestion
Now that you have CloudQuery and Elasticsearch running, you can configure the Elasticsearch destination. To do this, follow the instructions in the [Elasticsearch destination plugin docs](/docs/plugins/destinations/elasticsearch/overview) to create a configuration file called `elasticsearch.yml`.
```

File: website/pages/blog/search-through-cloud-infrastructure-with-elasticsearch.mdx [hunk 3]
```diff
@@ -0,0 +1,81 @@
...
+
+## Building an Historical View of Your Infrastructure
+
+One of the great things about Elasticsearch is that it is designed to be used with time series data, and therefore a final thing we wanted to note in this post is the ability to view historical snapshots of your infrastructure from Elasticsearch. The CloudQuery Elasticsearch destination currently supports three write modes: `overwrite`, `overwrite-delete-stale` (the default) and `append`. You can read more about these in the [plugin documentation](/docs/plugins/destinations/elasticsearch/overview), but in short, when using `append` mode, CloudQuery will never delete data, so you can build an historical view of your infrastructure over time. For example, by running a sync in `append` mode every day, you can build a visualization that shows the number of EC2 instances over time, or more generally track how (or whether!) your organization's security posture is improving over time.
```
Comment (erezrokah) [inline website/pages/blog/search-through-cloud-infrastructure-with-elasticsearch.mdx:77]:
```suggestion
One of the great things about Elasticsearch is that it is designed to be used with time series data, and therefore a final thing we wanted to note in this post is the ability to view historical snapshots of your infrastructure from Elasticsearch. The CloudQuery Elasticsearch destination currently supports three write modes: `overwrite`, `overwrite-delete-stale` (the default) and `append`. You can read more about these in the [plugin documentation](/docs/plugins/destinations/elasticsearch/overview#index-naming), but in short, when using `append` mode, CloudQuery will never delete data, so you can build an historical view of your infrastructure over time. For example, by running a sync in `append` mode every day, you can build a visualization that shows the number of EC2 instances over time, or more generally track how (or whether!) your organization's security posture is improving over time.
```

**Review by erezrokah — APPROVED** [review]
Looks great 🚀 A few non blocking comments

**Review by yevgenypats — APPROVED** [review]
Great blog!! 🚀  Love the charts and visualizations.

File: website/pages/blog/search-through-cloud-infrastructure-with-elasticsearch.mdx (hunk 2 above)
Comment (hermanschaaf) [reply website/pages/blog/search-through-cloud-infrastructure-with-elasticsearch.mdx]: Thanks, updated!

**Pull request merged**